skxray/tests/test_cdi.py

```diff
@@ -15,34 +15,20 @@ def dist_temp(dims):
     euclidian distance from array center.
     This is used for test purpose only.
     """
-    new_array = np.zeros(dims)
-
-    if np.size(dims) == 2:
-        x_sq = (np.arange(dims[0]) - dims[0]/2)**2
-        y_sq = (np.arange(dims[1]) - dims[1]/2)**2
-        for j in range(dims[1]):
-            new_array[:, j] = np.sqrt(x_sq + y_sq[j])
-
-    if np.size(dims) == 3:
-        x_sq = (np.arange(dims[0]) - dims[0]/2)**2
-        y_sq = (np.arange(dims[1]) - dims[1]/2)**2
-        z_sq = (np.arange(dims[2]) - dims[2]/2)**2
-        for j in range(dims[1]):
-            for k in range(dims[2]):
-                new_array[:, j, k] = np.sqrt(x_sq + y_sq[j] + z_sq[k])
-
-    return new_array
+    vec = [np.abs(np.arange(d) - (d-1.)/2.) for d in dims]
```

Comment:
This is the original code from Xiaojing. I just keep it here for comparison only. This code is not optimized, but is correct for sure.